application/src/test/java/com/sanction/thunder/authentication/ThunderAuthenticatorTest.java

```diff
@@ -1,54 +1,70 @@
 package com.sanction.thunder.authentication;
 
 import com.google.common.base.Optional;
-import com.google.common.collect.Lists;
 
+import com.sanction.thunder.dao.PilotUsersDao;
+import com.sanction.thunder.models.PilotUser;
 import io.dropwizard.auth.AuthenticationException;
 import io.dropwizard.auth.basic.BasicCredentials;
 
-import java.util.List;
-
 import org.junit.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
```

Comment:
These imports should be `org.junit.Assert.assertEquals` and not `junit.framework.TestCase.assertEquals`. Same for `assertFalse` and `assertTrue`.